.github/label-actions.yml

```
@@ -18,6 +18,10 @@
   comment: >
     We are using GitHub issues exclusively to track bugs in the core React Native library. Please try asking over on [Stack Overflow](http://stackoverflow.com/questions/tagged/react-native) as it is better suited for this type of question.
   close: true
+"Resolution: Community Repo":
+  comment: >
+    This issue can be better addressed in a different community repository. Please check out the list of [maintainers in the RNCommunity](https://github.com/react-native-community/.github/blob/master/MAINTAINERS.md) to find the best place to create this issue.
```

"can be better addressed" is going to often be too generous, as it's not like these issues could be fixed in this repo at all. How do you feel about this:
```suggestion
    This issue refers to a problem covered by a community module that should be tracked in that community repository. This repository only tracks bugs in the core React Native library. Please check out the list of [maintainers in the RNCommunity](https://github.com/react-native-community/.github/blob/master/MAINTAINERS.md) to find the best place to create this issue.
```

---

Yeah that sounds good, ill commit your edit.